shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -122,7 +122,9 @@ - (void)testItReportsPlatformBrightnessWhenViewWillAppear {
 }
 
 - (void)testItReportsDarkPlatformBrightnessWhenTraitCollectionRequestsIt {
-  if (!@available(iOS 13, *)) {
+  if (@available(iOS 13, *)) {
```

Comment:
Good catch. How did we miss this earlier? I expect clang had warnings for these.

Reply:
"-Werror" isn't on by default in xcode projects, we were previously using xcode to build them.